gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=org.ballerinalang
-version=0.6.9-SNAPSHOT
-ballerinaLangVersion=2.0.0-Preview5-SNAPSHOT
-stdlibIoVersion=0.5.1-SNAPSHOT
-stdlibLogVersion=1.0.1-SNAPSHOT
+version=0.7.0-SNAPSHOT
```

Comment:
Why are we bumping a minor version here? This should be `0.6.10-SNAPSHOT` right?